public/js/src/module/form-cache.js

```diff
@@ -466,7 +485,13 @@ function _updateCache( survey ) {
 
                         return formParts;
                     } )
-                    .then( _swapMediaSrc )
+                    .then( formParts => {
+                        if ( !isOnline ) {
+                            return _swapMediaSrc( formParts );
+                        }
+
+                        return formParts;
+                    } )
                     .then( _addBinaryDefaultsAndUpdateModel )
```

Comment:
Seeing all these steps makes me think that the survey does get cached even if online. What ensures that it does't?